test/LibraryManager.IntegrationTest/Services/InstallDialogTestService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.Test.Apex.Services;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Shell;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Web.LibraryManager.Vsix.UI;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest.Services
+{
+    /// <summary>
+    /// Test service for add client side libraries dialog.
+    /// </summary>
+    [Export(typeof(InstallDialogTestService))]
+    public class InstallDialogTestService : VisualStudioTestService
+    {
+        public InstallDialogTestExtension OpenDialog()
+        {
+            Guid guid = Guid.Parse("44ee7bda-abda-486e-a5fe-4dd3f4cefac1");
+            uint commandId = 0x0100;
+
+            Task.Factory.StartNew(() =>
+            {
+                this.UIInvoke(() =>
+                {
+                    this.CommandingService.ExecuteCommand(guid, commandId, null);
+                });
+            });
+
+            return this.WaitForDialog(TimeSpan.FromSeconds(5));
+        }
+
+        /// <summary>
+        /// Gets or sets the synchronization service reference.
+        /// </summary>
```

Comment:
Not really useful, especially since private. Would prefer to see our comment quota used on other areas of the code. :)

Reply:
Removed it in iteration 2.

---
In reply to: [205526798](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205526798) [](ancestors = 205526798)